src/components/Checkbox.tsx

```diff
@@ -48,6 +50,7 @@ const Checkbox: React.FC<Props> = (props) => {
         onFocus={onFocus}
         onBlur={onBlur}
         readOnly={readOnly}
+        form={form}
```

Comment:
why do you need this form parameter?

Reply:
Hi Hoan,

I understand your concerns regarding the form element and all.
I should have documented the <form/> problem+solution within the code (done that below).

The workaround with the empty <form/> element is needed because otherwise I had to nest a <form/> inside a <form/> which is illegal html.
Therefore I used the 'workaround' based on the second answer here:
https://stackoverflow.com/questions/3430214/form-inside-a-form-is-that-alright?lq=1

This does complicate the situation, but I really like to have the form for the colormap. Why?
Well first of all the colormap needs validation before it should be written back to the raster-layer form.
Creating this validation from scratch is hard and will also not be consistent with or other validation (which all use the form). Only doing this validation once the user submits the rasterlayer, would require us to re-open the modal (bad imo).

Second of all I think the Colormap is actually a form. Not only will this keep or code more consistent, it is also semantically correct. I know the backend does not threath the colormap as its own entity, but I think this is a 'implementation detail' on their side and I try to prevent copying such details to our code.

Hope this convinces you or otherwise feel free to comment question.
__________________________________________________________________________
I added the following comment to the code:
{/* 
      I also use a form inside the colormap modal.
      A form inside a form is not valid html.
      Therefore I used the workaround here:
      https://stackoverflow.com/questions/3430214/form-inside-a-form-is-that-alright?lq=1
      Answer from user: "ilevent"
      */}

Reply:
Ah ok. I understand your points. I am still not super convinced that we used a form inside a form. There must be a reason that a form nested inside another form is invalid in HTML. But your points are also valid so lets keep it this way.